.copr/rhc.spec.in

```diff
@@ -77,3 +77,123 @@ make PREFIX=%{_prefix} \
 
 
 %changelog
```

Comment:
I didn't know that I can cherry-pick anything from other repository. I will give a try.

Reply:
I can see that it is possible now, but I don't want to do such thing only because it is possible.

Reply:
I'm not sure I understand what you said.

The changelog is generated automatically, hence a static version a) is not useful b) would get stale pretty soon. Keeping it the sources would only be a waste, and I do not see how/what it would do good.